packages/playground/src/pages/edit.tsx

```diff
@@ -38,7 +38,8 @@ interface InternalTab extends Tab {
   _source: string;
   _name: string;
 }
-export const Edit = (props: { horizontal: boolean }) => {
+export const Edit = () => {
+  const [searchParams] = useSearchParams();
```

Comment:
useSearchParams / horizontal is pulled into the route, because with new router it's not allowed to use useSearchParams it outside the Router.